### PR TITLE
pmerge: add the pmerge package

### PR DESCRIPTION
**Pull request by downloadico**

### CODE REVIEW

**Comment by downloadico** [comment]
Can anyone tell me what went wrong here?

File: var/spack/repos/builtin/packages/pmerge/package.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    homepage = "https://github.com/beiko-lab/PMERGE"
+    git      = "https://github.com/beiko-lab/PMERGE.git"
+
+    version('master',  branch='master')
+
+    depends_on('automake@1.14:1.14.99', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('m4', type='build')
+    depends_on('perl', type='build')
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/pmerge/package.py:25]: The gitlab tests do not need to pass for this to be merged, but should this depend on `stacks` (actually looking at the PMERGE link, I think that would be `stacks@:1.48`).

Comment (downloadico) [reply var/spack/repos/builtin/packages/pmerge/package.py:25]: I'm not entirely clear!  
*My* reading of the pmerge page was that it *can* be used in a stacks pipeline.
It seems to build cleanly without stacks installed, but, again I'm not entirely clear on how it is supposed to work!
A user asked me to install it; I will check with them and see if they can give me some guidance on this.

Comment (downloadico) [reply var/spack/repos/builtin/packages/pmerge/package.py:25]: My user replied 'It should run independently of Stacks, but can be "inserted" into the stacks pipeline as an additional module compatible with stacks output down to version 1.48 or something like that.'.  Therefore, I don't think Stacks should be a dependency!

**Review by scheibelp — APPROVED** [review]

**Pull request merged**